odl/solvers/advanced/proximal_operators.py

```diff
@@ -524,9 +526,36 @@ def proximal_nonnegativity(space):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    If :math:`P` is the set of non-negative elements, the indicator function is
+    defined as
+
+        .. math::
+
+            I_{x \\in P} = \\left\{ \\begin{array}{ll}
```

Comment:
Rather `I_P(x)`, or `\\iota_P(x)` - the last notation seems to be rather wide spread for indicator functions.